.github/workflows/ci.yml

```diff
@@ -13,23 +13,20 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [14.x]
+        job: [lint, test]
 
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - uses: borales/actions-yarn@v2.0.0
-      with:
-        cmd: install --frozen-lockfile
-    - uses: borales/actions-yarn@v2.0.0
-      with:
-        cmd: build
-    - uses: borales/actions-yarn@v2.0.0
-      with:
-        cmd: test
-    - uses: borales/actions-yarn@v2.0.0
-      with:
-        cmd: lint
+    - name: install
+      run: yarn install --frozen-lockfile
+
+    - name: build
```

Comment:
Why run build before test and lint? Shouldn't the most lightweight job come first?

Reply:
We need to have built packages available in order to do some linting that checks types and the same goes for some of the tests. This is because this is a multi-package workspace that links the dependencies.